run_fastsurfer.sh

```diff
@@ -99,7 +100,8 @@ FLAGS:
                             installed already
   --sid <subjectID>       Subject ID to create directory inside \$SUBJECTS_DIR
   --sd  <subjects_dir>    Output directory \$SUBJECTS_DIR (or pass via env var)
-  --t1  <T1_input>        T1 full head input (not bias corrected). Requires an ABSOLUTE Path!
+  --t1  <T1_input>        T1 full head input (not bias corrected). Requires an
+                            ABSOLUTE Path!
```

Comment:
Why is this on a new line?

Reply:
Looks ok to me. Did you test if this works correctly in docker and native install?

Reply:
Newline because of line length